scripts/install_custom_dc.sh

```diff
@@ -63,17 +65,22 @@ ROOT=$PWD
 
 # Clone DC website repo and mixer submodule.
 if [[ ! -d "website" ]]; then
-  git clone https://github.com/datacommonsorg/website
+  git clone https://github.com/datacommonsorg/website --branch $CUSTOM_DC_RELEASE_TAG --single-branch
 fi
 
 cd website
-if [[ ! -d "mixer" ]]; then
-  git submodule foreach git pull origin master
-  git submodule update --init --recursive
-fi
+WEBSITE_GITHASH=$(git rev-parse --short=7 HEAD)
```

Comment:
Can we have a version file to store a fixed version file? And we can update it regularly.

Reply:
I agree, but I will leave that as a separate work because it is a bit involving (ex: we need to make sure it works with the version files in the separate source repo).